demo/src/app/components/toast/overview/toast-overview.component.html

```diff
@@ -0,0 +1,94 @@
+    To actually make it disappear, you can listen to the <a [routerLink]="['..', 'api']" fragment="NgbToast"><code>(hide)</code></a>
+    output and remove/destroy/hide it yourself.
+  </p>
+  <ngbd-code [snippet]="TOAST_INLINE_LIFECYCLE"></ngbd-code>
+  <br />
+  <p>Live example available <a [routerLink]="['..', 'examples']" fragment="inline" title="Declarative inline usage">here</a>.</p>
+</ngbd-overview-section>
+
+<ngbd-overview-section [section]="sections['toast-service']">
+  <p>Let's take the opportunity to demonstrate how to simply build a global toast management service.</p>
+  <ngb-alert [dismissible]="false" type="secondary">
+    <strong>TLDR;</strong>
+    You don't feel reading these long explanations? Go to the live example <a [routerLink]="['..', 'examples']" fragment="howto-global" title="Toast management service">here</a>.
+  </ngb-alert>
+
+  <p>In order to create our global toast system, 3 simple steps need to be done:<p>
+  <ol>
+    <li>Create a service <code>AppToastService</code>. Goal for this service is to act as a global storage.</li>
+    <li>Create a container component <code>&lt;app-toasts&gt;</code>, acting as the host in the application to display your toasts.
+    You could use <code>&lt;ngb-toast&gt;</code> with an <code>*ngFor</code> to read the list of toasts to display from the service.</li>
+    <li>Finally, use this container component in your application.</li>
+  </ol>
+
+  <h4>Global storage: DI to the rescue</h4>
+  <p>
+    Relying on Angular dependency injection to share some piece of logic application-wide is always a good and solid starting choice.
+  </p>
+  <p>
+    The service manages a collection of toasts. It also provides a public method to push a new toast to that same collection.
+    <ngbd-code [snippet]="APP_TOAST_SERVICE"></ngbd-code>
+  </p>
+
+  <ngb-alert [dismissible]="false" type="warning">
+    <svg:svg ngbdIcon="lightbulb" fill="currentColor" />
+    You could also create an interface to type your toast instead of using <code>any[]</code> here.
+  </ngb-alert>
+  <p>
+    Additionnally, a method to remove an existing toast from the collection is also implemented.
+    <ngbd-code [snippet]="APP_TOAST_SERVICE_REMOVE"></ngbd-code>
+  </p>
+
+  <h4>Creating a container component</h4>
+  <p>
+    As stated previously, <code>&lt;ngb-toast&gt;</code> only generate a valid Bootstrap toast markup.
+    Only rendering them anywhere would'nt make them positionned properly on the screen anyway.
+    <br />
+    Thus, as a suggestion, toasts could be rendered in the top right corner of the application, as a kind of oervlay.
+  </p>
+  <p>
+    To achieve that, you can create a dedicated component, acting as a container to render all toasts in a convenient way.
+    For example, this container could be positionned using CSS property <code>position: static</code>.
+  </p>
+  <ngb-tabset>
```

Comment:
I tried that. Single snippet is out of scope, has it is using 3 different languages. So it would be 3 snippets in a row. **much better with the tabset** trust me! 🤣